packages/sprotty/build/src/buildCodicon.ts

```diff
@@ -0,0 +1,72 @@
+import * as fs from 'fs';
+
+function transformCodicon() {
+    fs.readFile('../../node_modules/@vscode/codicons/dist/codicon.css', 'utf8', (err, cssData) => {
+        if (err) {
+            console.error(err);
+            return;
+        }
+
+        if (!fs.existsSync('./src/utils/generated')) {
+            fs.mkdirSync('./src/utils/generated');
+        }
+        if (!fs.existsSync('./css/generated')) {
+            fs.mkdirSync('./css/generated');
+        }
+
+        fs.readFile('../../node_modules/@vscode/codicons/dist/codicon.ttf', 'base64', (errTtf, dataTtf) => {
+            if (errTtf) {
+                console.error(errTtf);
+                return;
+            }
+            const ttfData = `data:font/ttf;base64,${dataTtf}`;
+            const alteredCssData = cssData.replace('"./codicon.ttf?0e5b0adf625a37fbcd638d31f0fe72aa"', ttfData);
+
+            fs.writeFileSync('./css/generated/codicon.css', alteredCssData);
```

Comment:
This is a redistribution of the codicon content and so we need to attribute that correctly according to their license:

https://github.com/microsoft/vscode-codicons/blob/e7d905a9cb4f06a0d4c4c93279967734faf66f0f/LICENSE#L217-L244

Reply:
Yes, thank you, I didn't check that, yet.